endpoints/makegoods.yaml

```diff
@@ -247,212 +247,71 @@ paths:
-              '{$request.body#/rejectCancelMakegoodResponseUrl}':
-                post:
-                  summary: Event from SGS that the create order request was rejected
-                  description: Event from SGS that the create order request was rejected
-                  requestBody:
-                    description: rejectCreateOrderResponseUrl callback
-                    required: true
-                    content:
-                      application/json:
-                        schema:
-                          $ref: 'schemas/commonSchemas.yaml#/components/schemas/encodededPayloadWrapper'
-                  responses:
-                    '202':
-                      description: Request processed successfully
-                      content:
-                        application/json:
-                          schema:
-                            $ref: 'schemas/commonSchemas.yaml#/components/schemas/acceptedResponse'
-
-  /v1/makegoods/unsubscribe:
-        delete:
-          summary: Unsubscribe to makegood events
-          description: Unsubscribe to makegood events
-          operationId: makegoodUnsubscribe
-          requestBody:
-            description: makegoodUnsubscribe
-            required: true
+          
+  /v1/makegoods/subscriptions/{id}:
+      delete:
+        summary: Remove a subscription
+        description: Existing subscriptions can be deleted by making a DELETE request against the subscription resource
+        operationId: Remove a subscription
+        parameters:
+        - name: id
+          in: path
+          description: The subscription id
+          required: true
+        responses:
+          '204':
+            description: Subscription deleted
+          default:
+            description: Unexpected error occured
             content:
               application/json:
                 schema:
-                  oneOf:
-                    - $ref: 'schemas/makegoodsSchemas.yaml#/components/schemas/makegoodsSubscribeSGS'
-                    - $ref: 'schemas/makegoodsSchemas.yaml#/components/schemas/makegoodsSubscribeSGSCounterParty'
-          responses:
-            '202':
-              description: Request processed successfully
-              content:
-                application/json:
-                  schema:
-                    $ref: 'schemas/commonSchemas.yaml#/components/schemas/acceptedResponse'
-            default:
-              description: Unexpected error occured
-              content:
-                application/json:
-                  schema:
-                    $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
\ No newline at end of file
+                  $ref: 'schemas/commonSchemas.yaml#/components/schemas/Error'
+      
+      get:
+        summary: Enumerating subscriptions
+        description: Get a list of active subscriptions
+        operationId: Enumerating subscriptions
+        parameters:
+        - name: id
```

Comment:
suggest removing id parameter. thought enumerate would return list of ids to facilitate deletes.